_1327/polls/models.py

```diff
@@ -54,6 +54,13 @@ def generate_new_title(cls):
 		return _("New Poll")
 
 	@classmethod
+	def get_vote_permission(klass):
```

Comment:
yeah that's right, but that method below is no classmethod, and a classmethod was necessary, that's why I added this